CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 #===============================================================================
 # Copyright 2020-2021 Intel Corporation
 # Copyright (C) 2022 Heidelberg University, Engineering Mathematics and Computing Lab (EMCL) and Computing Centre (URZ)
+# Copyright 2022 Codeplay Software
```

Comment:
You are sometimes adding Codeplay in some of the modified files but it is not consistent. We should check what the policy is for copyrights on this project. It would be easier if we don't have to add Codeplay copyrights to every file we modify but maybe only the files we create from scratch?

Reply:
I've remove the Codeplay copyright from modified files in [ed77b76](https://github.com/hjabird/oneMKL/pull/1/commits/ed77b76e4c380ecf167deb49999e81c63808517d).

The contribution guidelines for the oneMKL project do not provide guidance on adding copyrights to existing files, only that the file should start with a copyright.